tests/Php84/Php84Test.php

```diff
@@ -30,6 +30,34 @@ public function testMbLcFirst(string $string, string $expected): void {
         $this->assertSame($expected, mb_lcfirst($string));
     }
 
+    /**
+     * @dataProvider arrayFindDataProvider
+     */
+    public function testArrayFind(array $array, callable $callback, $expected): void {
```

Comment:
Don't we ensure our Symfony CS here in this repo?

Reply:
not automatically, I run php-cs-fixer from time to time for the moment